apps-rendering/src/liveBlock.ts

```diff
@@ -2,7 +2,7 @@
 
 import type { Block } from '@guardian/content-api-models/v1/block';
 import type { Tag } from '@guardian/content-api-models/v1/tag';
-import type { Option } from '@guardian/types';
+import { err, ok, OptionKind, partition, Result } from '@guardian/types';
```

Comment:
I think the linter wants you to separate out the import of `Result` into an `import type`.